analyzers/src/SonarAnalyzer.Common/Rules/WeakSslTlsProtocolsBase.cs

```diff
@@ -47,24 +48,27 @@ public abstract class WeakSslTlsProtocolsBase<TSyntaxKind, TIdentifierNameSyntax
 
         protected abstract GeneratedCodeRecognizer GeneratedCodeRecognizer { get; }
 
-        protected abstract TSyntaxKind SyntaxKind { get; }
+        protected abstract TSyntaxKind SyntaxKindOfInterest { get; }
```

Comment:
I would go for the simple name `SyntaxKind` since `OfInterest` does not add a lot of details.